finetune/base.py

```diff
@@ -296,17 +295,8 @@ def _distribute_strategy(self, visible_gpus):
         else:
             resolved_gpus = all_gpus()
 
-        num_gpus = len(resolved_gpus)
-        if num_gpus > 1:
-            distribute_strategy = PatchedParameterServerStrategy(
-                visible_gpus=resolved_gpus
-            )
-        elif num_gpus == 1:
-            gpu = resolved_gpus[0]
-            distribute_strategy = OneDeviceStrategy(device="/gpu:{}".format(gpu))
-        else:
-            distribute_strategy = OneDeviceStrategy(device="/cpu:0")
-
+        resolved_gpus_string = ['/gpu:{}'.format(gpu) for gpu in resolved_gpus]
+        distribute_strategy = tf.distribute.experimental.CentralStorageStrategy(resolved_gpus_string or None)
```

Comment:
Just read the docs for this -- its really nice that it just handles this by default now :)